SU-HSDO/suhumsci
# STN-602: Table heading link refinement

# READY FOR REVIEW

## Summary
Hover states for linked headings on tables did not have correct hover or focus states, which were not passing accessibility.
These linked headings did already have styling written but was not applying due to specificity, I have updated that class and it should now work as it should.

Example heading issue: https://creees-prod.stanford.edu/academicscapstone-thesis/past-theses
Jira Card: [STN-602](https://sparkbox.atlassian.net/browse/STN-602)

## Need Review By (Date)
11/16

## Urgency
medium

## Steps to Test
1. npm run test
2. A good page to use for testing is: http://sparkbox-sandbox.suhumsci.loc/publications/publication-table
3. Edit the view using the contextual link in the upper right hand of the view, above the filter across the right from the title, "Default Courses".
4. Go to Format > Settings
5. In the settings check any of the checkboxes to make the heading titles Sortable, then Apply, and then save the view.
6. The table view should now have links in those heading titles for sorting.

## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Sparkbox PR Checklist](../docs/SparkboxPRChecklist.md)
